src/tutorial/scheduling/integer-cycling.rst

```diff
@@ -345,11 +345,12 @@ Recurrence Sections
 
 .. ifnotslides::
 
-   In the previous examples we made the workflow repeat by placing the graph
-   in the ``P1`` setting. Here ``P1`` is a :term:`recurrence` meaning
-   repeat every cycle, where ``P1`` means every cycle, ``P2`` means every
-   *other* cycle, and so on. To build more complex workflows we can use multiple
-   recurrences:
+   From initial cycle point:
+      In the previous examples we made the workflow repeat by placing the graph
+      in the ``P1`` setting. Here ``P1`` is a :term:`recurrence` meaning
+      repeat every cycle, where ``P1`` means every cycle, ``P2`` means every
+      *other* cycle, and so on. To build more complex workflows we can use
+      multiple recurrences:
 
 .. code-block:: cylc
```

Comment:
The code block and image don't match the indentation
![image](https://user-images.githubusercontent.com/61982285/151579467-9ae4dc56-cffa-4829-ab64-abead4194e90.png)

Reply:
This is deliberate - I want them in the slide deck.

Reply:
Is there a way of at least increasing the space between the image and the next item "After arbitrary cycle point"?

Reply:
I think you can use a `|` character (which resets something in the doc state) to force a line break:

```rst
foo

|

bar
```

Reply:
I wasn't able to do this, but I put an  alt solution in place.